p2p/stream/common/requestmanager/requestmanager.go

```diff
@@ -349,26 +348,49 @@ func (rm *requestManager) pickAvailableStream(req *request) (*stream, error) {
 	return nil, errors.New("no more available streams")
 }
 
-func (rm *requestManager) addNewStream(st sttypes.Stream) {
+func (rm *requestManager) refreshStreams() {
 	rm.lock.Lock()
 	defer rm.lock.Unlock()
 
-	if _, ok := rm.streams[st.ID()]; !ok {
-		rm.streams[st.ID()] = &stream{Stream: st}
-		rm.available[st.ID()] = struct{}{}
+	added, removed := checkStreamUpdates(rm.streams, rm.sm.GetStreams())
+
+	for _, st := range added {
+		rm.logger.Info().Str("streamID", string(st.ID())).Msg("add new stream")
+		rm.addNewStream(st)
+	}
+	for _, st := range removed {
+		rm.logger.Info().Str("streamID", string(st.ID())).Msg("remove stream")
```

Comment:
may change to debug level , IMO.

Reply:
These log messages may appear a lot in an unstable network (which is the case of error injected in stn) where peer connection is unstable thus a lot of connect/disconnect operation is executed. For stable network, this shall not appear a lot. Please let me know if this log appears a lot in a stable network.